### DISCUSSION TITLE
Weapons

### DISCUSSION THREAD

**Discussion by ghost** [WeAreMahsaAmini/Core#248] [Ideas]
The people need weapons if they're actually going to succeed against the regime. One thing I keep hearing from people in Iran is that the government has only been trying to suppress the movement with the Police or Anti-Riot teams. This scares me because as we all know, "Sepah" is the main force that they have in defending the regime and these guys haven't stepped onto the streets yet.

The people of Iran are indeed numerous, and courageous and they won't give up easily. But in the absence of the army, they need some sort of physical force to be able to actually overcome the ruthless vultures. 

Before this movement, I was against guns. But I think in this particular situation, access to guns is what our people need. Any ideas on how we can establish a logistical chain to do this?

P.S. I think the reason they did a drone strike in Iraqi Kurdistan today, was in an attempt to stop a channel like this.

**Comment by syavash** [comment]
@ejtempo I'm going to lock this conversation since it's promoting violence. The mission of this idea is not aligned with what we do.
